### DISCUSSION TITLE
Automatically update plugins

### DISCUSSION THREAD

**Discussion by Traap** [LazyVim/LazyVim#548] [Q&A] [answered]
I'm sure I've missed this in another post or documentation.

How do I accomplish something similar to the bash snipped below for Neovim powered by lazy.nvim?

#!/bin/bash
vim +PlugUpdate
nvim +PackerSync

**Comment by loiccoyle** [comment, marked as answer]
I use `nvim --headless '+Lazy! sync' +qa`

**Reply by Traap** [reply]
Thanks.  I knew I had read something similar.
